app/src/main/kotlin/com/wire/android/ui/userprofile/common/UserProfileInfo.kt

```diff
@@ -0,0 +1,143 @@
+    isEditable: Boolean = false,
+    onEditClick: (() -> Unit)? = null,
+) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center,
+        modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentHeight()
+            .padding(top = dimensions().spacing16x)
+    ) {
+        UserProfileAvatar(
+            onClick = onUserProfileClick,
+            isClickable = if (!isEditable) false else !isLoading,
+            size = dimensions().userAvatarDefaultBigSize,
+            avatarAssetByteArray = avatarAssetByteArray,
+            status = UserStatus.NONE,
+        )
+        if (isLoading) {
+            Box(
+                Modifier
+                    .padding(MaterialTheme.wireDimensions.userAvatarClickablePadding)
+                    .clip(CircleShape)
+                    .background(MaterialTheme.wireColorScheme.onBackground.copy(alpha = 0.7f))
+            ) {
+                WireCircularProgressIndicator(
+                    progressColor = MaterialTheme.wireColorScheme.surface,
+                    modifier = Modifier.align(Alignment.Center)
+                )
+            }
+        }
+    }
+    ConstraintLayout(
+        Modifier
+            .fillMaxWidth()
+            .wrapContentHeight()
+    ) {
+        val (userDescription, editButton, teamDescription) = createRefs()
+
+        Column(
+            modifier = Modifier
+                .padding(horizontal = dimensions().spacing64x)
+                .wrapContentSize()
+                .constrainAs(userDescription) {
+                    top.linkTo(parent.top)
+                    bottom.linkTo(parent.bottom)
+                    start.linkTo(parent.start)
+                    end.linkTo(parent.end)
+                }
+        ) {
+            Text(
+                modifier = Modifier.align(Alignment.CenterHorizontally),
+                text = fullName,
+                overflow = TextOverflow.Ellipsis,
+                maxLines = 1,
+                style = MaterialTheme.wireTypography.title02,
+                color = MaterialTheme.colorScheme.onBackground,
+            )
+            Text(
+                modifier = Modifier.align(Alignment.CenterHorizontally),
+                text = "@$userName",
+                overflow = TextOverflow.Ellipsis,
+                style = MaterialTheme.wireTypography.body02,
+                maxLines = 1,
+                color = MaterialTheme.wireColorScheme.labelText,
+            )
+        }
+
+        if (isEditable) {
+            IconButton(
+                modifier = Modifier
+                    .padding(start = dimensions().spacing16x)
+                    .constrainAs(editButton) {
+                        top.linkTo(userDescription.top)
+                        bottom.linkTo(userDescription.bottom)
+                        end.linkTo(userDescription.end)
+                    },
+                onClick = onEditClick!!,
```

Comment:
Is there a special reason behind, to let this parameter being nullable ? I think it's possible that a NPE happens here 🤔